### PR TITLE
Wifi_changes

### PR DESCRIPTION
**Pull request by Snowflake6**
Updating to allow multiple WiFi configurations

### CODE REVIEW

File: src/EPDWifi.h [hunk 1]
```diff
@@ -21,22 +21,31 @@
...
    WiFi.disconnect();
    WiFi.setAutoConnect(true);
    WiFi.setAutoReconnect(true);
 
-   Serial.print("Connecting to ");
-   Serial.println(WIFI_SSID);
-   delay(100);
-   
-   WiFi.begin(WIFI_SSID, WIFI_PW);
+/* add AP's here with AP Name, AP Password pairs */
```
Comment (wizche) [inline src/EPDWifi.h:38]: I would not just add plain credentials here but use the same mechanism as before. Aka in the `ConfigTemplate.h` you add for example two or three entries:
```cpp
#define WIFI_SSID1       "your wifi ssid"
#define WIFI_SSID1_PWD       "your wifi pwd"
#define WIFI_SSID2       "your wifi ssid"
#define WIFI_SSID2_PWD       "your wifi pwd"
```
and then we can use the `ifdef` construct for compilation. Like
```cpp
#ifdef WIFI_SSID1 
wifiMulti.addAP(WIFI_SSID1 , WIFI_SSID1_PWD);
#endif
#ifdef WIFI_SSID2
wifiMulti.addAP(WIFI_SSID2 , WIFI_SSID2_PWD);
#endif
```

File: src/Display.h [hunk 2]
```diff
@@ -146,11 +146,10 @@ float getBtcValue()
    }
 }
 
-/* Draw a the head with version, city, rssi and battery */
+/* Draw a the head with AP Name, city, rssi and battery */
 void WeatherDisplay::DrawHead()
 {
-
-   canvas.drawString("BTC/USD " + String(getBtcValue()), 20, 10);
```
Comment (wizche) [inline src/Display.h:153]: Can you please keep the BTC/USD price ticker? Maybe move it to the right or move the AP to the right...

File: .vscode/extensions.json [hunk 3]
```diff
@@ -1,7 +1,10 @@
...
-        "platformio.platformio-ide"
-    ]
-}
+{
+    // See http://go.microsoft.com/fwlink/?LinkId=827846
+    // for the documentation about the extensions.json format
+    "recommendations": [
+        "platformio.platformio-ide"
+    ],
+    "unwantedRecommendations": [
```
Comment (wizche) [inline .vscode/extensions.json:7]: Can you remove this unwanted entry from the extension.json?

**Review by wizche — CHANGES_REQUESTED** [review]
Congrats on your first PR @Snowflake6 !
Can you please address these comments so I can merge the branch?
Thanks a lot

File: src/EPDWifi.h (hunk 1 above)
Comment (Snowflake6) [reply src/EPDWifi.h:38]: Yeah, I really, really didn't want to do that as it's such an inelegant way to code this.  I really wanted a #define construct that would let me define an array of access points, but I couldn't figure out how to do that.  I'll implement the suggestion, and allow up to 10 access points.

**Comment by Snowflake6** [comment]
All changes requested have been implemented.
- WiFi config moved back to ConfigTemplate.h, up to 10 AP's possible.
- Made top left corner configurable in ConfigTemplate.h.  Other options could be added.
- extensions.json updated as requested - this may be due to compiling this on both Ubuntu and Win10.

**Pull request merged**